decidim-surveys/lib/tasks/surveys/upgrade/decidim_surveys_upgrade_tasks.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :decidim_surveys do
+  namespace :upgrade do
+    desc "Migrates the component permission from respond to response"
+    task fix_survey_component_permissions: :environment do
```

Comment:
We should do the same for `Decidim::ResourcePermission` and rename this task name in the process:

> decidim-development-app(dev)> Decidim::ResourcePermission.pluck :permissions
>   Decidim::ResourcePermission Pluck (0.6ms)  SELECT "decidim_resource_permissions"."permissions" FROM "decidim_resource_permissions"
> => [{"answer"=>{"authorization_handlers"=>{"id_documents"=>{}, "postal_letter"=>{}}}}]


```suggestion
    desc "Migrates the component permission from answer to respond"
    task fix_survey_permissions: :environment do
```

Reply:
funny, i have not seen this previous time. 
It should be fixed now ( without spec on resource permission) as the `Decidim::ResourcePermission` doesn't  have a factory bot.

Reply:
@andreslucena this should be fixed by [4c5b9e4](https://github.com/decidim/decidim/pull/14940/commits/4c5b9e4515f099a0ec130252acee5be9d6448f6d)